header.php

```diff
@@ -1,9 +1,10 @@
 <!DOCTYPE html>
 <html <?php language_attributes(); ?>>
-<head>
-	<meta charset="utf-8">
-	<meta name="viewport" content="width=device-width">
-	<title>Setupshop0.1</title>
+  <head>
+	  <meta charset="utf-8">
+	  <meta name="viewport" content="width=device-width">
+    <img src="C:\Bitnami\wampstack-5.6.25-0\apache2\htdocs\Setupshop\wp-content\uploads\2016\09">
```

Comment:
Your src is a direct link, as soon as you publish this online it will break. You shouldn't not be referring anything in your uploads folder directly in your theme. Create an img folder in your theme for that. 
Secondly you cannot use img tags within your head.